superset-frontend/src/dashboard/components/nativeFilters/FilterBar/CascadeFilters/CascadeFilterControl/index.tsx

```diff
@@ -53,28 +49,26 @@ const CascadeFilterControl: React.FC<CascadeFilterControlProps> = ({
   onFilterSelectionChange,
 }) => (
   <>
-    <StyledFilterControlBox>
-      <StyledCaretIcon name="caret-down" />
-      <FilterControl
-        dataMaskSelected={dataMaskSelected}
-        filter={filter}
-        directPathToChild={directPathToChild}
-        onFilterSelectionChange={onFilterSelectionChange}
-      />
-    </StyledFilterControlBox>
-
-    <StyledCascadeChildrenList>
+    <FilterControl
+      dataMaskSelected={dataMaskSelected}
+      filter={filter}
+      directPathToChild={directPathToChild}
+      onFilterSelectionChange={onFilterSelectionChange}
+    />
+    <StyledDiv>
       {filter.cascadeChildren?.map(childFilter => (
-        <li key={childFilter.id}>
+        <>
+          <Icons.CaretDownOutlined iconSize="s" />
```

Comment:
I personally think the caret is not needed here, as it is just a static component that occupies a fair amount of vertical space and doesn't really add a lot of information.

Reply:
I'm not against removing the query, it was my first design choice. The order can be enough to show hierarchy. About the indentation though, I really think that is not a good solution to keep inputs with different widths. Also, it will break if the user adds a bunch o nesting.